R/series-s3-waldo.R

```diff
@@ -2,7 +2,7 @@
 compare_proxy.polars_series <- function(x, path) {
   list(
     object = structure(
-      x$to_r_vector(ensure_vector = FALSE, int64 = "character", decimal = "character"),
+      x$to_r_vector(ensure_vector = FALSE, int64 = "integer", decimal = "character"),
```

Comment:
Sorry, this wasn't my intention, instead of modifying the contents of `compare`, could you cast the column types of the expected DataFrame to match?
Something like `pl$DataFrame(x = c(6, 5, 2))$cast(x = pl$Int64)`

Reply:
Done